docs/Form.md

```diff
@@ -47,6 +47,7 @@ Here are all the props you can set on the `<Form>` component:
 * [`id`](#id)
 * [`noValidate`](#novalidate)
 * [`onSubmit`](#onsubmit)
+* [`sanitizeEmptyValues`](#sanitizeemptyvalues)
```

Comment:
Upgrade guide still says `sanitizeEmptyValues` prop has been removed -> we should update that

Reply:
Right, fixed